pkg/sql/publisher.go

```diff
@@ -62,6 +62,12 @@ func NewPublisher(db db, config PublisherConfig, logger watermill.LoggerAdapter)
 		logger = watermill.NopLogger{}
 	}
 
+	if config.AutoInitializeSchema && isTx(db) {
+		// either use a prior schema with a tx db handle, or don't use tx with AutoInitializeSchema
+		return nil, errors.New("tried to use AutoInitializeSchema with a database handle that looks like" +
```

Comment:
it may be nice to refer here to `AutoInitializeSchema` and add more info in `AutoInitializeSchema`

Reply:
e423e77b58ea678d7dc9673a92b762ee6186e2da expanded the `AutoInitializeSchema` godoc